src/main/java/org/openstreetmap/atlas/checks/flag/CheckFlag.java

```diff
@@ -451,6 +469,22 @@ public int hashCode()
         return new MultiIterable<>(getShapes()).iterator();
     }
 
+    /**
+     * Decouple the {@link CheckFlag} from any
+     * {@link org.openstreetmap.atlas.geography.atlas.Atlas}s by making all the
```

Comment:
nit: Could we do a static import here?

Reply:
It would not be static, as Atlas is a class. I could import the class, but that seems excessive for just a doc reference.